### DISCUSSION TITLE
Interconnect remote support settings

### DISCUSSION THREAD

**Discussion by MSHAFFER64** [HewlettPackard/POSH-HPOneView#617] [Q&A]
Is there a OneView Cmdlet that will get the remote support configuration for a interconnect? 
Get-OVserver | Get-OVRemoteSupportSetting  -> {WORKS}
Get-OVEnclosure | Get-OVRemoteSupportSetting  -> {WORKS}
Get-OVInterconnect | Get-OVRemoteSupportSetting  -> {Not supported}
I know there is API but is there Cmdlets to do it?

**Comment by ChrisLynchHPE** [comment]
The Get-OVRemoteSupportSetting Cmdlet is supposed to support retrieving the configured remote support settings.  What specifically isn't working?  Is there an error message?  What version of the library are you using?
